tests/console/libvorbis.pm

```diff
@@ -36,4 +36,8 @@ sub run {
     assert_script_run("ogginfo sample.ogg 2>&1 | grep -Pz '(?s)(?=.*Processing file \"sample.ogg\")(?=.*Channels: 2)(?=.*Rate: 44100)(?=.*Total data length: 15170 bytes)(?=.*Playback length: 0m:00.751s)(?=.*Average bitrate: 161.418024 kb/s)'");
 }
 
+sub post_run_hook {
+    # Do not clear the console
```

Comment:
I'm afraid you will need a lot of validation runs as you touch a lot of tests not only on public cloud - and I only see validation runs for public cloud

Reply:
I am already running QAM Updates tests for all supported distributions. Do you have some other suggestions?

Reply:
well, every scenario using libvorbis.pm - the change here is strange anyway. Possibly you're fixing something for some other test here

Reply:
You're right. My problem was caused by the previous failing test. I discarded the change here. New verification runs are in progress.